modules/dataframe/src/processors/csvsource.cpp

```diff
@@ -133,4 +156,92 @@ void CSVSource::deserialize(Deserializer& d) {
+                    itemFilters.push_back(csvfilters::stringMatch(
+                        detail::getValue<IntProperty>(cp, "column"),
+                        detail::getValue<TemplateOptionProperty<filters::StringComp>>(cp, "comp"),
+                        detail::getValue<StringProperty>(cp, "match")));
+                } else if (startsWith(identifier, "intItem")) {
+                    itemFilters.push_back(csvfilters::intMatch(
+                        detail::getValue<IntProperty>(cp, "column"),
+                        detail::getValue<TemplateOptionProperty<filters::NumberComp>>(cp, "comp"),
+                        detail::getValue<IntProperty>(cp, "value")));
+                } else if (startsWith(identifier, "floatItem")) {
+                    itemFilters.push_back(csvfilters::floatMatch(
+                        detail::getValue<IntProperty>(cp, "column"),
+                        detail::getValue<TemplateOptionProperty<filters::NumberComp>>(cp, "comp"),
+                        detail::getValue<FloatProperty>(cp, "value"),
+                        detail::getValue<FloatProperty>(cp, "epsilon")));
+                } else if (startsWith(identifier, "doubleItem")) {
+                    itemFilters.push_back(csvfilters::doubleMatch(
+                        detail::getValue<IntProperty>(cp, "column"),
+                        detail::getValue<TemplateOptionProperty<filters::NumberComp>>(cp, "comp"),
+                        detail::getValue<DoubleProperty>(cp, "value"),
+                        detail::getValue<DoubleProperty>(cp, "epsilon")));
+                } else if (startsWith(identifier, "intRangeItem")) {
+                    const auto& range = detail::getValue<IntMinMaxProperty>(cp, "range");
+                    itemFilters.push_back(csvfilters::intRange(
+                        detail::getValue<IntProperty>(cp, "column"), range.x, range.y));
+                } else if (startsWith(identifier, "floatRangeItem")) {
+                    const auto& range = detail::getValue<FloatMinMaxProperty>(cp, "range");
+                    itemFilters.push_back(csvfilters::floatRange(
+                        detail::getValue<IntProperty>(cp, "column"), range.x, range.y));
+                } else if (startsWith(identifier, "doubleRangeItem")) {
+                    const auto& range = detail::getValue<DoubleMinMaxProperty>(cp, "range");
+                    itemFilters.push_back(csvfilters::doubleRange(
+                        detail::getValue<IntProperty>(cp, "column"), range.x, range.y));
+                }
+            }
+        }
+    };
+
+    csvfilters::Filters filters;
+    parse(includeFilters_, filters.includeRows, filters.includeItems);
+    parse(excludeFilters_, filters.excludeRows, filters.excludeItems);
+
+    if (!rowComment_.get().empty()) {
+        filters.excludeRows.push_back(csvfilters::rowBegin(rowComment_, true));
+    }
+    if (!keepOnly_.get().empty()) {
```

Comment:
I would guess that this use case is rather uncommon.
Having “keep top N”/“keep range”, as a special quick access filter would in my opinion be more useful.

Reply:
I left it in there as it is the opposite of comment rows.